01_Provisioning/src/module/api.ts

```diff
@@ -69,71 +72,54 @@ export class ProvisioningModuleApi extends AbstractModuleApi<ProvisioningModule>
         itemsPerMessageSetVariables = itemsPerMessageSetVariables == null ?
             setVariableData.length : itemsPerMessageSetVariables;
 
-        // We don't wait for each message to be responded to, instead this is fire-and-forget.
-        // Only a failure from the message bus to the central system is caught, downstream failures are invisible.
-        // Users can pull these variables from the data endpoint to check results afterwards.
-        // Max wait time before being able to view results should be:
-        // (setVariableData.length / itemsPerMessageSetVariables) * this.config.websocketServer.maxCallLengthSeconds
-        // TODO: Consider adding a callback url to response to check results
-        const finalMessageConfirmation: IMessageConfirmation = { success: true };
         while (setVariableData.length > 0) {
-            const setVariableDataSubset = setVariableData.slice(0, itemsPerMessageSetVariables);
-            const messageConfirmation = await this._module.sendCall(identifier, tenantId, CallAction.SetVariables,
-                { setVariableData: setVariableDataSubset } as SetVariablesRequest);
-            if (!messageConfirmation.payload) {
-                finalMessageConfirmation.success = false;
-                finalMessageConfirmation.payload = finalMessageConfirmation.payload ?
-                    (finalMessageConfirmation.payload as SetVariableDataType[]).concat(setVariableDataSubset) : setVariableDataSubset
-            }
+            await this._module.sendCall(identifier, tenantId, CallAction.SetVariables,
```

Comment:
@thanaParis what if this `sendCall` fails? We seem to only return `success` here?